### ISSUE TITLE
UI for easy creation / monitoring of pipelines

### ISSUE THREAD

**Issue by fabiante** [redpanda-data/connect#581]
Hi @Jeffail,

tldr: love this project, would love to get involved and use Benthos a lot more in my projects. Ever thought about adding a basic web UI for managing and monitoring pipelines?

Long version: I am following Benthos for quiet a while now. I really enjoy seeing releases that almost always add stuff that I could use in projects I work at. Not only that, I think Benthos is a good example of a modern open source project that focuses on functionality and gets the job done. The website, the documentation and last but not least the "marketing" done via Youtube videos etc. is on point.

That said, the company I work at is often lacking a general-purpose data conversion / processing software that maps data from one format onto another. I have already worked at some companies and have seen lot's of proprietary tools mapping JSON <-> XML, manipulating some properties, filtering some data entries and so on.

I would like to introduce Benthos to my colleagues because I think it is a viable option to deal with a lot of challenges we face without writing another proprietary tool again, there is enough of those 😉

The thing that keeps me from going in that direction is that Benthos lacks a GUI. I am actually not "afraid" of the CLI and configuration files but sadly a lot of people need a graphical representation to understand what is going on.

Now I wonder, have you ever considered adding a GUI? My guess is that development of Benthos at its current stage focuses on features, support for more components and "internal behind the scenes stuff" rather than development of a user interface. If so, I think thats absolutely fine. IMHO a working software with "only" a CLI and config files > a buggy, unstable software with a fancy web UI. But still, I'd like to know what your opinion on this topic is. Are there considerations to add this in the future? What if *someone* would be willing to add a UI? No promises but I could imagine that we may do so if Benthos became our go-to tool for data integration. Same goes for adding components or processors. 😉 

Looking forward to your thoughts. ✌️

**Comment by Jeffail** [comment]
Hey @FabianTe, this is definitely something I'd love to introduce and makes total sense for the Benthos project as whole. Personally, my main focus is probably always going to be on improving the foundational service as that's where I'm most productive, but if someone were to attempt this then I'd be more than willing to lend some help.

Occasionally if I get burned out I'll hack together a POC for something related and that will sometimes lead towards a real thing (like https://lab.benthos.dev, or Bloblang), so there's a chance I'd get around to this at some point but my existing TODO list right now is pretty long.

**Comment by twmb** [comment]
Closing as part of a backlog cleanup. Pipeline management UI work is now done in Redpanda Console — feature requests for UI functionality can be made there. If there's renewed interest in something specific to Connect, feel free to open a new issue or PR.
